apps/web/src/data/subgraph/requests/auctionHistory.ts

```diff
@@ -0,0 +1,40 @@
+import * as Sentry from '@sentry/nextjs'
+
+import { SDK } from 'src/data/subgraph/client'
+import { CHAIN_ID } from 'src/typings'
+
+import { Auction_OrderBy, OrderDirection } from '../sdk.generated'
+
+export type AuctionHistory = {
```

Comment:
can we use the AuctionHistoryQuery type generated from the sdk instead of making an extra type